chcemvediet/apps/accounts/migrations/0005_socialapp_remove_twitter_and_linkedin.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.contrib.sites.models import Site
+from django.db import models, migrations
+
+
+def forward(apps, schema_editor):
+    SocialApp = apps.get_model(u'socialaccount', u'SocialApp')
+    SocialAccount = apps.get_model(u'socialaccount', u'SocialAccount')
+    SocialApp.objects.filter(provider__in=[u'twitter', 'linkedin']).delete()
```

Comment:
@martinmacko47 preferujes moznost `provider__in` alebo `Q` expressions?

Reply:
@viliambalaz Pri jednoduchych porovnaniach je asi lepsie `__in` ako si spravil. Ak by to boli nejake zlozite vyrazy, vtedy by bolo asi lepsie `Q`, nech mas nad tym vacsiu kontrolu.